askama/src/filters/mod.rs

```diff
@@ -336,6 +336,54 @@ pub fn wordcount<T: fmt::Display>(s: T) -> Result<usize> {
     Ok(s.split_whitespace().count())
 }
 
+pub struct DisplaySome<'a, T>(Option<&'a T>);
+
+impl<T> fmt::Display for DisplaySome<'_, T>
+where
+    T: fmt::Display,
+{
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        if let Some(val) = self.0 {
+            write!(f, "{val}")?;
+        }
+        Ok(())
+    }
+}
+
+pub fn display_some<T>(value: &Option<T>) -> Result<DisplaySome<'_, T>>
```

Comment:
Please put the filter functions before the types that need it.

Can we make `DisplaySome` and `DisplaySomeOr` private and yield `impl Display` from the filter functions?